docky/cmd/run_open.py

```diff
@@ -20,10 +20,9 @@ class DockyExec(DockySub):
     service = cli.SwitchAttr(["service"])
 
     def _use_specific_user(self, service):
-        user = self.project.get_user(service)
         if self.root:
             user = "root"
-        return user
+        return self.project.get_user(service)
```

Comment:
This breaks root right?

Reply:
aahahha my finger didn't do what my brand was thinking !!
It's ok now